packages/pieces/community/pdf-to-text/src/lib/actions/pdf-to-text.ts

```diff
@@ -0,0 +1,24 @@
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { readPdfText } from 'pdf-text-reader';
+
+export const parseTextFromPdfFile = createAction({
+  // auth: check https://www.activepieces.com/docs/developers/piece-reference/authentication,
+  name: 'parseTextFromPdfFile',
+  displayName: 'Parse text from PDF file',
+  description: 'Parses texts from PDF file',
+  props: {
+    file: Property.LongText({
+      displayName: 'PDF File URL',
+      required: true,
+    }),
+  },
+  async run(context) {
+    // Action logic here
+    console.dir(context, { depth: null });
```

Comment:
Nit: please remove this, we shouldn't log props value they might contains sensitive values